docs/cloud/managing-airbyte-cloud.md

```diff
@@ -47,6 +47,22 @@ To delete a workspace:
 2. In the Workspace Settings sidebar, click **General Settings**.
 3. Click **Delete your workspace**.
 
+## Single Workspace vs. Multiple Workspaces
+ 
+You can use one or more workspaces with Airbyte Cloud. When deciding how many workspaces you need, some important factors to consider are user access and billing.
+ 
+### Access: Benefits and Considerations
+| Number of Workspaces | Benefits                                                                      | Considerations                                                                                                                              |
```

Comment:
@Amruta-Ranade , I used the table generator to make these tables, but I can't seem to get their formatting right. Any suggestions for better formatting?
<img width="859" alt="Screen Shot 2022-07-11 at 5 43 40 PM" src="https://user-images.githubusercontent.com/106352739/178385734-d49bbd8f-6de3-45b3-9efd-fc9dc15d8b4c.png">
?

Reply:
Looks okay to me. What do you want to change?

Reply:
I thought that it was strange that the Benefits column was so much smaller than the Considerations column, and that the columns in the first table don't line up with the columns in the second table.

Reply:
Gotcha. Yeah that's one of the quirks of Markdown tables. You can fiddle with them and google how to resize columns (might have to use html instead of Markdown) but not worth the time and effort IMO.

Reply:
Sounds good! I will leave it be :)